backends/cadence/aot/TARGETS

```diff
@@ -28,6 +28,7 @@ python_library(
         "compiler.py",
     ],
     deps = [
+        "fbsource//third-party/pypi/pyre-extensions:pyre-extensions",
```

Comment:
Is Buck target used here or meant not to export?

Reply:
OSS uses both buck and CMake I believe for now